docs/teal-basics.md

```diff
@@ -0,0 +1,121 @@
+###### tags: `tealgarden` `ethturin`
+
+# Basic Explainers Teal Garden Track :books:
+
+**Hackers**, welcome to our quick explainers, to give you a conception of what you need to consider building with us at **ETH Turin**.
+
+Don't feel overwhelmed by all the materials, we simply want to give you a little primer to get started more easily :)
+
+## What is holacracy?
+
+**Holacracy** is a new way of structuring and running your organization that replaces the conventional management hierarchy. Instead of operating top-down, power is distributed throughout the organization – giving individuals and teams freedom while staying aligned to the organization’s purpose.
+
+**Holacracy-powered organizations** focus on purpose at every level of scale: organizational purpose, team purpose, and individual purpose are all explicit and aligned. Every team member directs their energy in alignment with the broader mission, unlocking your organization’s full potential.
+
+## What is a Teal Organization?
+
+Characterized by **self-management, wholeness, and a deeper sense of purpose**, these organizations – which are described as **“Teal”** – operate largely without organization charts, management hierarchies, quarterly goals or other traditional management strategies.
+
+Instead, they’re characterized by features like **self-managed teams**, intuitive reasoning and **decentralized decision-making**. Because these practices are so new, the conversation around what they are and how to implement them is **ongoing**.
+
+## What is a process ?
+
+A simple Wikipedia entry can help us understand what a process is:
+
+> “A business process or business method is a collection of related, structured activities or tasks by people or equipment in which a specific sequence produces a service or product (serves a particular business goal) for a particular customer or customers.”
+
+We created an evaluation page to make sure we & you know what a good process is all about:
+
+- [What is a valid submission to teal.garden?](https://hackmd.io/dMlEYrGOTIm8Yw4e50V8qA?view)
+
+## What is a component ?
+
+The components are the building blocks of our process. Most of the time, a simple list is sufficient, to make sure we have all requirements in place.
+
+A component can be **a role, a policy, or a tool**. They can be exisiting in the organizations setup, or will be added during the process. All these preparing investigations help understand the dynamics of the components and applying them in a new setup.
+
+**Roles & Policies are connected to People.
+Tools are connected to Technology.**
+
+Anyone who wants to adopt your process will need to follow the instructions to setup a tool. It consist of one or several steps. Each step decribes how to integrate one tool into the process. Make sure to document each component which is related to a tool.
```

Comment:
maybe move all this to instructions. just say: if you have a component that is a tool, you will have to give instructions how to use that tool in the `instructions` section.